File: Codigo_Fonte_c/Lista 3/lista3_1.c
#include <stdio.h>

int main(){
	char nome[10][20];
	int i;
	for(i = 0; i < 10; i++){
		scanf("%s", nome[i]);
	}
	printf("A lista de nome:\n");
	for (i = 0; i < 10; i++){
		printf("%s\n", nome[i]);

	}
	return 0;	
}

File: Codigo_Fonte_pascal/lista3/lista3_1.pas
program le_vetor;

var
   nome : array[1..10] of string;
   i : integer;

begin
     for i := 1 to 10 do
         readln(nome[i]);

     writeln('A lista de nome:');
     for i:= 1 to 10 do
         writeln(nome[i]);
     readln;

end.
